internal/results/archiver.go

```diff
@@ -38,14 +44,14 @@ func getFileTarHeader(file string) (*tar.Header, error) {
 }
 
 // Creates a zip file in the outputDir containing each file in the filePaths slice.
-func CompressResultsArtifacts(outputDir string, filePaths []string) error {
+func CompressResultsArtifacts(outputDir string, filePaths []string) (string, error) {
```

Comment:
Please update func's comment to include this new string output param.

Reply:
Yup I can do that.  👍